pages/common/hut.md

```diff
@@ -0,0 +1,29 @@
+# hut
+
+> A CLI tool for sourcehut.
+> Note: Use `hg` in place of `git` below if using Mercurial. See `man hut` for additional details.
+> More information: <https://man.sr.ht>.
```

Comment:
https://man.sr.ht is for sourcehut itself, not `hut`.
```suggestion
> More information: <https://manned.org/hut>.
```

Reply:
Yeah and sourcehut's page on hut says... read man hut for more.  Do we absolutely have to have a link?  Because the man page is always the up to date source and what the online source explicitly defers to.

Reply:
Every tldr page has a "More information" link. https://manned.org/hut is the man page, so I'm not sure what you're referring to.

Reply:
@mbsmith Manned exact does that. It lists local man page of distributions online. In this case, we can go with it as it lists the latest man page of hut rather than my previous suggestion which doesn't yet.